password_enhancements.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Password Enhancements'
+description: 'Password related enhancements.'
+package: Security
+type: module
+core: 8.x
+php: 7.1
```

Comment:
It should be `"7.1"` otherwise module cannot be enabled on the UI.
![image](https://user-images.githubusercontent.com/1755573/60582830-d6b86580-9d89-11e9-9b4b-ec437f4a73c4.png)

Reply:
https://www.drupal.org/project/drupal/issues/2934098

Reply:
Omg :D I haven't noticed, thanks :)